### ISSUE TITLE
Shut off daily Scan

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#15383]
## Description
The daily accessibility scan is currently only monitored by the Sitewide accessibility lead (Laura). However, it's not as efficient or as effective as it could be. We have decided to turn it off for now and revisit this later when AP is further along and we can look at building this better.

## Acceptance Criteria
- [x] Daily scan no longer runs
- [x] Slack notifications for scan are turned off
- [ ] Discuss with Erika about updating [Platform documentation](https://depo-platform-documentation.scrollhelp.site/developer-docs/daily-accessibility-scan-for-va-gov-static-content)

## Team
Please check the team(s) that will do this work.

- [x] CMS Team
- [ ] Public Websites
- [ ] Facilities
- [ ] User support
- [ ] Accelerated Publishing

**Comment by laflannery** [comment]
@ariperez I just realized that I'm not getting the pings for this which has been so nice so thank you so much for that! However I wanted to circle back to the final AC on this - the Platform documentation. Have you spoken to Erika about this yet or should I open a new ticket for us to look at this?

**Comment by laflannery** [comment]
@EWashb and @BerniXiongA6 FYI I finished the 3rd AC here. The Platform documentation has been updated with the platform team: https://depo-platform-documentation.scrollhelp.site/developer-docs/daily-accessibility-scan-for-va-gov-static-content
